package.json

```diff
@@ -13,9 +13,13 @@
   "repository": "github:SAP/cloud-sdk",
   "scripts": {
     "test": "npm run test:unit && npm run test:integration && npm run test:type",
-    "test:unit": "npx lerna run test --ignore \"@sap-cloud-sdk/integration-tests\" --ignore \"@sap-cloud-sdk/type-tests\"",
+    "test:unit": "npx lerna run test --ignore \"@sap-cloud-sdk/integration-tests\" --ignore \"@sap-cloud-sdk/type-tests\" --ignore \"@sap-cloud-sdk/e2e-tests\"",
     "test:integration": "npx lerna run test --scope \"@sap-cloud-sdk/integration-tests\"",
     "test:type": "npx lerna run test --scope \"@sap-cloud-sdk/type-tests\"",
+    "test:e2e": "npx lerna run test --scope \"@sap-cloud-sdk/e2e-tests\"",
+    "e2e-test:build": "cd test-packages/e2e-tests && npm run deploy:to-sqlite",
+    "e2e-test:start-server": "cd test-packages/e2e-tests && npm start",
```

Comment:
the command `e2e-test:start-server` is not used in the the build pipeline. I guess it is for local testing. Perhaps it makes sense to include a `test-local:start` and `test-local:watch` here and link it to the two scripts in the e2e-tests/package.json. Also I would like to split the generate block in more  things: generate-vdm-test-entties-v2,
generate-vdm-test-entties-v4, generate-e2e-service and generate-all (naming is super bad, just to get the four different things clear.

Reply:
I would remove the watch command, since for such a small app, you don't need it and it asks for additional dependency: '@sap/cds-dk'.

Reply:
ok